examples/connectivity/plot_ica_resting_state.py

```diff
@@ -13,6 +13,11 @@
 nyu_dataset = datasets.fetch_nyu_rest(n_subjects=1)
 func_filename = nyu_dataset.func[0]
 
+# print basic information on the dataset
+print('First anatomical nifti image (3D) is at: %s' % nyu_dataset.anat_anon[0])
+print('First functional nifti image (4D) is at: %s' %
+      nyu_dataset.func[0])  # 4D data
```

Comment:
You are right, but it depends on 'n_subjects' argument above. To also make
the datasets clearer I would therefore suggest to leave the printings as
they are and increase the #subjects to 2 instead. The aim is educational.

2015-04-13 10:36 GMT+02:00 Loïc Estève <notifications@github.com>:

> In examples/connectivity/plot_ica_resting_state.py
> <https://github.com/nilearn/nilearn/pull/531#discussion_r28221872>:
>
> > @@ -13,6 +13,11 @@
> >  nyu_dataset = datasets.fetch_nyu_rest(n_subjects=1)
> >  func_filename = nyu_dataset.func[0]
> >
> > +# print basic information on the dataset
> > +print('First anatomical nifti image (3D) is at: %s' % nyu_dataset.anat_anon[0])
> > +print('First functional nifti image (4D) is at: %s' %
> > +      nyu_dataset.func[0])  # 4D data
>
>  I don't think it should say first here since the list is of size 1,
> maybe something like 'Functional 4D image is at: '
>
> This is where my earlier comment was supposed to end up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/531/files#r28221872>.
>



-- 
Viele Grüße,
Danilo